src/modules/aws.rs

```diff
@@ -127,6 +170,13 @@ pub fn module<'a>(context: &'a Context) -> Option<Module<'a>> {
         return None;
     }
 
+    // only display if credential_process is defined or has valid credentials
```

Comment:
Should we also support the [environment variable](https://docs.aws.amazon.com/sdkref/latest/guide/setting-global-credential_process.html) e.g `AWS_SECRET_ACCESS_KEY` and `AWS_SESSION_TOKEN`?